indexer/services/ender/src/lib/translated-types.ts

```diff
@@ -1,7 +1,34 @@
-import { IndexerAssetPosition, IndexerPerpetualPosition, IndexerSubaccountId } from '@dydxprotocol-indexer/v4-protos';
+import { Liquidity } from '@dydxprotocol-indexer/postgres';
+import {
+  IndexerAssetPosition,
+  IndexerOrder,
+  IndexerPerpetualPosition,
+  IndexerSubaccountId, LiquidationOrderV1,
+} from '@dydxprotocol-indexer/v4-protos';
+import { Long } from '@dydxprotocol-indexer/v4-protos/build/codegen/helpers';
 
```

Comment:
nit: Maybe a comment here stating these are the canonical object types for handling messages from the protocol in `ender`.

Reply:
done